protobufs/gen/go/transactions/service.pb.go

```diff
@@ -6,13 +6,12 @@ package transactionspb
 import (
 	context "context"
```

Comment:
why are these being regenerated? only entity should have been regenerated, right?